### PR TITLE
Differentiate mangroves and other aquatic veg

### PR DESCRIPTION
**Pull request by emmaai**
Now `mangroves=124` and `others=125` in level 1

### CODE REVIEW

File: odc/stats/plugins/l34_utils/l4_natural_aquatic.py [hunk 1]
```diff
@@ -5,400 +5,282 @@
...
             "b": veg_cover,
-            "c": lifeform,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
-    l4 = expr_eval(
-        "where((a==124)&(b==12)&(c==2), 81, a)",
+    ##########################################
```
Comment (tebadi) [inline odc/stats/plugins/l34_utils/l4_natural_aquatic.py:245]: as above

File: odc/stats/plugins/l34_utils/l4_natural_aquatic.py [hunk 2]
```diff
@@ -5,400 +5,282 @@
...
+            "a": res,
             "b": veg_cover,
-            "c": lifeform,
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
+    #########################################
```
Comment (tebadi) [inline odc/stats/plugins/l34_utils/l4_natural_aquatic.py:204]: as above

File: odc/stats/plugins/l34_utils/l4_natural_aquatic.py [hunk 3]
```diff
@@ -5,400 +5,282 @@
...
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
-
-    l4 = expr_eval(
-        "where((a==124)&(b==13)&(c==2)&(d==2), 86, a)",
+    ##########################################
```
Comment (tebadi) [inline odc/stats/plugins/l34_utils/l4_natural_aquatic.py:164]: same as above

File: odc/stats/plugins/l34_utils/l4_natural_aquatic.py [hunk 4]
```diff
@@ -5,400 +5,282 @@
...
-            "d": water_seasonality,
         },
-        name="mark_cultivated",
+        name="mark_final",
         dtype="uint8",
     )
 
-    l4 = expr_eval(
-        "where((a==124)&(b==10)&(c==2)&(d==2), 80, a)",
+    #########################################
```
Comment (tebadi) [inline odc/stats/plugins/l34_utils/l4_natural_aquatic.py:124]: unwanted artefact

**Review by tebadi — APPROVED** [review]

File: odc/stats/plugins/l34_utils/l4_natural_aquatic.py (hunk 4 above)
Comment (emmaai) [reply odc/stats/plugins/l34_utils/l4_natural_aquatic.py:124]: it's not unwanted.  It's meant to be there to section the code

> unwanted artefact

**Pull request merged**